assets/adept.yaml

```diff
@@ -47,3 +47,24 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+- type: model
+  name: Fuyu
+  organization: Adept
+  description: Fuyu is a small version of the multimodal model that powers our product.
```

Comment:
Change to make more natural
"Fuyu is a small version of the multimodal model that powers Adept's core product."